exercises/change/example.go

```diff
@@ -0,0 +1,119 @@
+// Package change demonstrates making change with fewest number of coins.
+package change
+
+// Change returns a list of coins in increasing value order to make
+// change for target amount from the coins given; an infinite supply of coins is assumed.
+func Change(coins []int, target int) (result []int) {
+	solutions := make([][]int, 0)
+	if target == 0 {
+		result = []int{}
+	} else {
+		if target > 0 {
+			findSolutions(coins, target, &solutions)
```

Comment:
I was a little confused by this, maybe a comment that makes it clearer that `findSolutions()` updates `solutions`? Or it could just be me being dense! :smile: